boards/ble400/dist/openocd.cfg

```diff
@@ -0,0 +1,5 @@
+source [find interface/jlink.cfg]
+transport select swd
```

Comment:
Honestly speaking I have no experience in OpenOCD configs but comparing to other nrf51 based boards it it `hla_swd`. Can you give me a quick hint about that line please?